content/applications/websites/website/configuration/translate.rst

```diff
@@ -2,111 +2,138 @@
-the required languages and add them to your website. To do so, go to :menuselection:`Website -->
-Configuration --> Settings` and click :icon:`fa-arrow-right` :guilabel:`Install new languages` in
-the :guilabel:`Website Info` section. In the dialog box that opens, select the :guilabel:`Languages`
-you want from the dropdown menu, tick the required :guilabel:`Websites to translate`, and click
-:guilabel:`Add`.
-
-To edit your website's languages, go to :menuselection:`Website -–> Configuration -–> Settings` and
-add/remove the required languages in/from the :guilabel:`Languages` field in the
-:guilabel:`Website info` section.
+To allow translation on the website, first :doc:`install <../../../general/users/language>`
+the required languages. To do so, go to :menuselection:`Website -->
+Configuration --> Settings`.
+And, under the :guilabel:`General` section, click :icon:`oi-arrow-right`
+:guilabel:`Install new languages`. In the dialog box, select the :guilabel:`Languages`
+from the dropdown menu, click :guilabel:`Add`, and click :guilabel:`Close`.
 
-.. tip::
-   Alternatively, once the languages have been installed, you can add them from the :ref:`language
-   selector <translate/language-selector>`. You might then need to refresh your page to see the new
-   language.
+Then, add the language to the website by clicking on the :guilabel:`Languages` field, selecting the
+desired language and clicking :guilabel:`Save`.To remove a language from the website, click the
+:icon:`oi-close` :guilabel:`(close)` next to it.
 
 .. _translate/default-language:
 
-Default language
-----------------
-
-When multiple languages are available on your website, you can set a default language to be used if
-the visitor’s browser language is not available. To do so, go to :menuselection:`Website –->
-Configuration -–> Settings`, and select a language in the :guilabel:`Default` field.
-
-.. note::
-   This field is only visible if multiple languages have been installed and added to your website.
+If the visitor’s browser language is not available, the website will display its content in the
+database’s :ref:`default language <translate/default-language>`. To edit the default language, go to
+:menuselection:`Website --> Configuration --> Settings`. Under the :guilabel:`General` section,
+select the desired language in the :guilabel:`Default` field.
 
 .. _translate/language-selector:
 
 Language selector
 =================
 
-Your website’s visitors can switch languages using the language selector, available by default in
-the :guilabel:`Copyright` section at the bottom of the page. To edit the language selector menu:
+Once other languages are installed and added to the website, a :guilabel:`Language selector`
+appears on the website, allowing the switch between languages. It automatically appears in
+the header as a dropdown menu and in the :guilabel:`Copyright` section at the very bottom of the
+page.
```

Comment:
```suggestion
Once other languages are added to the website, a language selector is displayed in the website's header and footer, allowing visitors to switch between languages.
```
- I believe it can be shortened/simplified. Specifying it's in the footer's "copyright" section isn't really useful at this stage, for example, even if it's right.
- No :guilabel: for language selector (it's a bit confusing when to use guilabels or not, but in this case, we refer to the "concept" of language selector, not to a field/element where the name "language selector" would be visible on screen.